src/routes/(console)/+layout.svelte

```diff
@@ -274,6 +276,8 @@
             $stripe = await loadStripe(VARS.PUBLIC_STRIPE_KEY);
             await checkForMissingPaymentMethod();
         }
+
+        identifyUserWithReo();
```

Comment:
I think we just need that once in a Layout of authenticated part of the console, that way we only have to call it once